src/main/java/seedu/address/model/person/Age.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's age in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAge(String)}
+ */
+public class Age {
+
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Age should only contain numbers between 0 and 149 both inclusive.";
+    public static final String VALIDATION_REGEX = "\\b[0-9]|[1-9][0-9]|1[0-4][0-9]\\b";
+    public final String value;
+
+    /**
+     * Constructs a {@code Age}.
+     *
+     * @param age A valid phone number.
+     */
+    public Age(String age) {
+        requireNonNull(age);
+        checkArgument(isValidAge(age), MESSAGE_CONSTRAINTS);
+        value = age;
+    }
+
+    /**
+     * Returns true if a given string is a valid phone number.
```

Comment:
I think you meant to write age here, similar for the header comment of the class constructor above.

Reply:
Good catch! Updated in the latest commit.